README.md

```diff
@@ -52,11 +52,48 @@ ${1:document}.addEventListener('${2:event}', function(e) {
 });
 ```
 
+### [oc] onclick
+
+```javascript
+${1:document}.onclick = function(e) {
+	${0:// body...}
+};
+```
+
+### [ol] onload
+
+```javascript
+${1:document}.onload = function(e) {
+	${0:// body...}
+});
+```
+
+### [of] onfocus
+
+```javascript
+${1:document}.onfocus = function(e) {
+	${0:// body...}
+});
+```
+
+### [ob] onblur
+
+```javascript
+${1:document}.onblur = function(e) {
+	${0:// body...}
+});
```

Comment:
The significant drawback with inline events is that unlike event listeners, you may only have one inline event assigned. Inline events are stored as an attribute/property of the element, meaning that it can be overwritten.

We recommend you to use `addEventListener` instead.